### PR TITLE
Issue34/fix Loading URL with Electron instead of React

### PR DESCRIPTION
**Pull request by jthvjn**
Loading the remote ide (che on minishift) to the electron browser window.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.9/angular.min.js"></script>
```
Comment (benoitf) [inline public/index.html]: we shouldn't use remote resources

File: src/components/App/WorkspaceDashboard/WorkspaceDashboard.tsx [hunk 2]
```diff
@@ -37,18 +38,20 @@ class WorkspaceDashboard extends React.Component<any,any>{
...
             </div>
         )
     }
-    private getMinishiftIp()
-    {
-        return '';
-    }
+     private getMinishiftIp()
+     {
+         return '192.168.42.177';
```
Comment (benoitf) [inline src/components/App/WorkspaceDashboard/WorkspaceDashboard.tsx]: it looks it's overlapping with 

```
createWindow() {
mainWindow.loadURL('http://che-mini-che.192.168.42.177.nip.io/dashboard/#/');

```

File: public/electron.js [hunk 3]
```diff
@@ -6,12 +6,13 @@ const BrowserWindow = electron.BrowserWindow;
...
 const isDev = require('electron-is-dev');
+const ipcMain = electron.ipcMain;
 
 let mainWindow;
 
 function createWindow() {
-  mainWindow = new BrowserWindow({width: 900, height: 680});
-  mainWindow.loadURL(isDev ? 'http://localhost:3000' : `file://${path.join(__dirname, '../build/index.html')}`);
+  mainWindow = new BrowserWindow({width: 1500, height: 950});
+  mainWindow.loadURL('http://che-mini-che.192.168.42.177.nip.io/dashboard/#/');
```
[the remark this answers is not in the document]
Comment (waveywaves) [reply public/electron.js]: We have been using this method for some time now and hasn't given any problems but this one time :/

[the remark this answers is not in the document]
Comment (waveywaves) [reply public/electron.js]: screenshot from 2018-10-09 13-33-25

This is the error, and is expecting us to load angular modules.

**Comment by waveywaves** [comment]
Closing as the issue has been resolved

**Pull request closed without merging**